CIGI-969: update article_type of article series article pages

yang-wang-ds opened the pull request:
#### Description of changes
resolves CIGIHub/cigi-tickets#969

- use `/api/article_series_article_pages/` to extract a summary of article series article pages and their types before and after migration
- migrate to apply change
- what changes do we have to make regarding the search table?

#### Pull Request checklist
This checklist needs to be completed before assigning reviewers. If the checklist will not be completed, please comment with the reason.
- [x] Have you reviewed your own code changes?
- [x] Has the issue owner reviewed your changes?
- [x] Have you given the PR a descriptive title?
- [x] Have you described your changes above? Always include screenshots if applicable.
- [x] Have you pushed your latest commit to this branch?

---
#### Code Review checklist
This checklist should be completed if applicable before approving the pull request.
- [ ] Have you reviewed the code changes?
- [ ] Have you tested the changes on Google Chrome?
- [ ] Have you tested the changes on Safari?
- [ ] Have you tested the changes on Microsoft Edge (non-Chromium)?
- [ ] Have you tested the changes on iOS?
- [ ] Have you tested the changes on Android?

johnbxu commented:
The testing database doesn't have the "Essay" type, so it's breaking the tests. Should add a pass condition so if it can't find the article type it doesn't do anything.

johnbxu commented:
RE: search tables, for example the search table in cigionline/static/pages/article_landing_page/index.js will only grab article pages that are Interviews, Op-Eds, or Opinion types. We will need to add Essay to the list for them to display there. 
However, after thinking about it more, it occurs to me that we might need to display the Essays on the publications landing page (cigionline/static/pages/publication_list_page/index.js) instead of the article landing page. That will require adding article pages to the types displayed, as well as adding the Essay sub type to the filters.
Also need to look at the main search page to make sure the essay type will display properly, that there is a filter for it, and that the icon displays properly - should use the publication icon I guess.

yang-wang-ds commented:
migration:
- added check whether "Essay" as an ArticleTypePage already exists
- if not, create this as a child page to ArticlesListPage

search tables:
- added "Essay" type to opinions and publications search tables
- set "Essay" results icon to opinion icon in topbar search table
